docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,629 @@
+
+![submit_pr_squash_commits_7.png](./imgs/submit_pr_squash_commits_7.png)
+
+This can give a user a chance to view all messages before forumating one final message.
+
+If I had a large number of commits and didn't want all commit messages to be appended together, I would have specified: `pick, fixup, fixup` instead of `pick, squash, squash` . The command `fixup` is like `squash`, except that it discards a commit's log message.
+
+In my case, I only like the message of the first commit, so I will delete the others.
+
+Press `i` to place the editor into `INSERT` mode, and prepare the single commit message.
+
+![submit_pr_squash_commits_8.png](./imgs/submit_pr_squash_commits_8.png)
+
+To save these changes, press the `escape` key, and type:
+
+```
+:wq
+```
+
+The interactive `rebase` command finally completes.
+
+![submit_pr_squash_commits_9.png](./imgs/submit_pr_squash_commits_9.png)
+
+
+Now it is time to push the changes from the local feature branch to the remote feature branch.
+
+Type :
+
+```
+git push -f
+```
+
+You can confirm that there is only 1 commit by typing:
+
+```
+git log
+```
+
+![submit_pr_squash_commits_11.png](./imgs/submit_pr_squash_commits_11.png)
+
+or via your browser in your remote feature branch.
+
+![submit_pr_squash_commits_10.png](./imgs/submit_pr_squash_commits_10.png)
+
+
+### 17. Creating the pull request (PR)
+
+Now that you have completed the `rebase` and `commit squashing` steps, it is finally time to create the pull request.
+
+In your browser, navigate to your remote `feature` branch, and press the `Compare & pull request` button.
+
+![submit_pr_create_pr_1.png](./imgs/submit_pr_create_pr_1.png)
+
+The short and long description fields of the pull request are auto-filled using the short and long description of your commit.
+
+![submit_pr_create_pr_2.png](./imgs/submit_pr_create_pr_2.png)
+
+Fill in as much extra information as possible to properly describe the purpose of the pull request ( for example, fixing a specific bug or implementing an extension point, etc).
+
+(The title of the pull request should start with “WIP” for “work in progress” when the pull request is not complete yet.)
+
+ Press the `Create pull request` button.
+
+
+
+![submit_pr_create_pr_3.png](./imgs/submit_pr_create_pr_3.png)
+
+The pull request is created, and some continuous integration jobs commence.
+
+![submit_pr_create_pr_4.png](./imgs/submit_pr_create_pr_4.png)
+
+Make certain all these jobs complete successfully.
+
+### 18. Agreeing to the contributor license agreement (CLA)
+
+You must agree to the contributor license agreement (CLA) before the pull request can be `merged`. Please read
+[Agreeing to the CLA](https://loopback.io/doc/en/contrib/code-contrib.html#agreeing-to-the-cla).
+
+### 19. Go through the PR review process
+
+Request a code review or document review.
+
+Add a specific committer to speed up this process. (for example, @bajtos, @raymondfeng).
```

Comment:
Perhaps you could link to their accounts (e.g. `[@bajtos](https://github.com/bajtos)`) ?

Reply:
tbh, I'm not so sure about adding specific maintainer just to speed up the process. :) 
For each package, there is a list of codeowners specified in https://github.com/strongloop/loopback-next/blob/master/CODEOWNERS. They are the maintainers who have deeper knowledge in the specified package and the default reviewers of the PR.  
I think we can put it as... if you think there are other contributors should be reviewing your PR, you can add them as one of the reviewers.

Reply:
I was just repeating what is being said in 
https://loopback.io/doc/en/contrib/code-contrib.html
![image](https://user-images.githubusercontent.com/6864736/52651718-b19a8780-2eba-11e9-89d5-0bdfeb472055.png)

Reply:
So should I instead state the a contributor should look into CODEOWNERS file and pick someone from there?

Reply:
Code owners will automatically be requested for review

Reply:
oh, very cool.